Cargo.toml

```diff
@@ -100,6 +101,7 @@ native-tls = { version = "0.2", optional = true }
 postgres-native-tls = { version = "0.5", optional = true }
 refinery = { version = "0.8", features = ["tokio-postgres"], optional = true }
 serde_json = { version = "1.0", optional = true }
+tempdir = { version = "0.3", optional = true }
```

Comment:
?

Reply:
No idea where that came in from, I'm removing it...